plugins/@grouparoo/iterable/src/lib/export/exportProfile.ts

```diff
@@ -0,0 +1,77 @@
+import { ExportProfilePluginMethod } from "@grouparoo/core";
+import { connect } from "../connect";
+import { addToList, removeFromList } from "./listMethods";
+
+export const exportProfile: ExportProfilePluginMethod = async ({
+  appGuid,
+  appOptions,
+  export: {
+    toDelete,
+    newProfileProperties,
+    oldProfileProperties,
+    newGroups,
+    oldGroups,
+  },
+}) => {
+  // if we received no mapped data... just exit
+  if (Object.keys(newProfileProperties).length === 0) {
+    return { success: true };
+  }
+  const client = await connect(appOptions);
+
+  const email = newProfileProperties["email"]; // this is how we will identify profiles
+  if (!email) {
+    throw new Error(`newProfileProperties[email] is a required mapping`);
+  }
+
+  try {
+    let user;
+    try {
+      user = await client.users.get({ email });
+    } catch (error) {
+      if (!error.toString().match(/Request failed with status code 404/)) {
+        throw error;
+      }
+    }
+    if (toDelete) {
+      if (user) {
```

Comment:
if we don't have the user, can use just `deleteByEmail` and be sure to have a test where the user doesn't actually exist, so we can handle that.